config/routes.rb

```diff
@@ -22,6 +22,7 @@
       post "kit_items_only", to: "kit_items#create_kit_items_only"
       patch "kit_items_only/:id", to: "kit_items#update_kit_items_only"
       get "profile", to: "users#profile"
+      get "users/profile", to: "users#profile"
```

Comment:
The ticket instructions say the endpoint is users/profile, but it was actually just /profile, so you didn't need to add another route to the same place. Just remove this line.